german-iso.lbx

```diff
@@ -11,6 +11,14 @@
                   {betreut von}},
   urlalso      = {{auch verf\"{u}gbar unter}
                   {auch verf\"{u}gbar unter}},
+  director     = {{Direktor}% female: Direktorin
```

Comment:
"Direktor" is wrong for the director if a film. The German term would be "Regisseur" for a male and "Regisseurin" for a female. Depending on how the term appears in the output, a gender neutral alternative (that even feels more natural in many situations) would be "Regie".